.github/workflows/tests.yml

```diff
@@ -27,6 +27,22 @@ jobs:
         uses: actions/dependency-review-action@v3
         if: github.event_name == 'pull_request'
 
+      # XXX: `make static-analysis` also run shellcheck but this one provides
```

Comment:
What is "xxx" here for?

Reply:
![Screenshot from 2023-09-26 14-40-31](https://github.com/canonical/lxd/assets/545509/5c46d97b-2599-4a58-ab1f-7a44a353a7ae)
https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

Reply:
How bizarre. OK 😀